cns/restserver/api.go

```diff
@@ -1148,19 +1180,16 @@ func (service *HTTPRestService) publishNetworkContainer(w http.ResponseWriter, r
 			defer publishResponse.Body.Close()
 		}
 
-		// Store ncGetVersionURL needed for calling NMAgent to check if vfp programming is completed for the NC
-		primaryInterfaceIdentifier := getInterfaceIdFromCreateNetworkContainerURL(req.CreateNetworkContainerURL)
-		authToken := getAuthTokenFromCreateNetworkContainerURL(req.CreateNetworkContainerURL)
-
 		// we attempt to extract the wireserver IP to use from the request, otherwise default to the well-known IP.
 		hostIP := extractHostFromJoinNetworkURL(req.JoinNetworkURL)
```

Comment:
should remove this from all of these paths

Reply:
removed